packages/react/src/utils/scriptLoader.ts

```diff
@@ -95,6 +100,14 @@ export async function loadScript(params: LoadScriptParams): Promise<HTMLScriptEl
     if (proxyUrl) {
       script.setAttribute('data-clerk-proxy-url', proxyUrl);
     }
+    if (domain) {
+      script.setAttribute('data-clerk-domain', domain);
+    }
+
+    if (isSatellite) {
+      script.setAttribute('data-clerk-is-satellite', isSatellite ? 'true' : 'false');
```

Comment:
:upside_down_face: 
```suggestion
      script.setAttribute('data-clerk-is-satellite', String(!!isSatellite));
```

Reply:
Now that I am thinking about it, maybe isSatellite should no be there. Instead I it should be included in the options of the `load` function. This is because isSatellite is required only after a `Clerk` instance already exists.